airbyte-api/src/main/openapi/config.yaml

```diff
@@ -317,6 +322,66 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/SourceDefinitionReadList"
+  /v1/source_definitions/list_opt_in:
+    post:
+      tags:
+        - source_definition
+      summary: List all opt-in sourceDefinitions along with the current workspace's opt-in statuses
```

Comment:
I think I'm not totally happy with the term 'opt-in' now that I see it here in code. Maybe we'd be better off calling these 'private' instead of 'opt-in'. Especially because the boolean that controls this is called 'public', I think 'private' is a more obvious counterpart. Also opt-in is a bit misleading, since customers aren't actually able to opt themselves in, they need to be granted access by an admin. 

Also I think the 'summary' field should mention something about needing a 'grant', just to help readers understand the tie-in between this route and the 'grant' model that we're introducing

What do you think?

Reply:
Yeah i agree that I don't really like the term 'opt-in' either.  I didn't put much thought into the naming previously but I agree that using 'private' will read much better.
Also +1 to a better summary.